aggregator/aggregator.go

```diff
@@ -107,6 +118,11 @@ func NewAggregator[Input any, Output any](
 	}, nil
 }
 
+// Start method runs the main loop of the Aggregator. This loop has 2 main events:
+//   - Get a response from the BLS aggregation service: In this case the response is processed and sent to
+//     the Task Manager on-chain contract.
+//   - Receive a new task created event log: In this case the aggregator processes that event, and sends to
+//     the bls aggregation service the new task created metadata.
```

Comment:
This includes too many implementation details.

```suggestion
// Starts running the Aggregator. This should be called only one time per Aggregator.
//
// The main loop of the Aggregator has 2 main events:
//   - Get a response from the BLS aggregation service: In this case the response is processed and sent to
//     the Task Manager on-chain contract.
//   - Receive a new task created event log: In this case the aggregator processes that event, and sends to
//     the bls aggregation service the new task created metadata.
```